lib/moon_web/pages/design/button/icon_button_page.ex

```diff
@@ -69,15 +69,11 @@ defmodule MoonWeb.Pages.Design.Button.IconButtonPage do
   def render(assigns) do
     ~F"""
     <Page {=@theme_name} {=@active_page} {=@breadcrumbs} {=@direction}>
-      <ComponentPageDescription title="IconButton">
+      <ComponentPageDescription
+        title="IconButton"
+        image="/moon/assets/images/facing/components/icon_button.png"
```

Comment:
i'd preffer to see common part of image pathes inside ComponentPageDescription module, and here - only "components/icon_button.png"
the same for ComponentsPageCard & ComponentsPageSection modules.

Reply:
I think full path is more explicit and works also well (because of IDE and search-replace) + enables more dynamic/any path for the image path. (Let's say you have 1 image, what you want to use in 3 places.)

Reply:
looks reasonable. @tkullisaar please start from facing. e.g. here should be "facing/components/icon_button.png"